### DISCUSSION TITLE
Is it possible to strip SNI?

### DISCUSSION THREAD

**Discussion by a-vad** [mitmproxy/mitmproxy#7173] [Q&A] [answered]
I am trying to strip the Server Name Indication (SNI) from the traffic that goes through my proxy. I can unfortunately not find any documentation on how to achieve that. Can this be done?

**Comment by emanuele-em** [comment, marked as answer]
Hi! Yes, you can use `flow.server_conn.sni` [[Doc](https://docs.mitmproxy.org/stable/api/mitmproxy/flow.html)] and more specifically [[server_conn doc](https://docs.mitmproxy.org/stable/api/mitmproxy/connection.html#Server)] :)

**Reply by a-vad** [reply]
Thanks! I will check it out.
